server/routes/handlers/projects.js

```diff
@@ -49,6 +49,19 @@ module.exports = {
     (req, res) => res.status(200).json(res.locals.project),
   ],
 
+  patchStatus: [
+    (req, res, next) =>
+      projectController
+        // Save by ensuring the payload contains the proper id which is used for perms
+        .update(req.app.locals.project, { status: req.body.status }, { id: req.params.id })
```

Comment:
Should we perhaps limit this to only the two available statuses? Especially if we introduce approval later on.

Also, whenever it comes to matter, I think we should used `confirmed` as our status rather than `verified` which you have in your tests.